features/desktop/single_post_view_moderation.feature

```diff
@@ -7,52 +7,29 @@
       | bob        |
       | alice      |
     And a user with username "bob" is connected with "alice"
+    And "bob@bob.bob" has a public post with text "Here is a post to test with"
```

Comment:
I optimized that file by creating the post directly, not using the view. We won 11 seconds ;)